lua/telescope/actions/init.lua

```diff
@@ -143,14 +143,21 @@ function actions.toggle_selection(prompt_bufnr)
   current_picker:toggle_selection(current_picker:get_selection_row())
 end
 
+local get_scroll_speed = function(prompt_bufnr)
```

Comment:
I am thinking maybe we should move this to `action_set`. Similar to `set.shift_selection = function(prompt_bufnr, change)` and all we need to pass in is direction (-1 or 1)

Reply:
Otherwise LGTM. Thanks for picking it up :)